lib/manageiq/environment.rb

```diff
@@ -16,14 +16,17 @@ def self.manageiq_plugin_update(plugin_root = nil, force_bundle_update: true)
       # determine plugin root dir. Assume we are called from a 'bin/' script in the plugin root
       plugin_root ||= Pathname.new(caller_locations.last.absolute_path).dirname.parent
 
+      ENV["SKIP_DATABASE_RESET"] ||= "true" if ENV["CI"]
+
+      ensure_config_files
+
+      puts "== Installing dependencies =="
       setup_gemfile_lock if ENV["CI"]
       install_bundler(plugin_root)
       bundle_config(plugin_root)
       bundle_update(plugin_root, force: force_bundle_update)
 
-      ensure_config_files
-
-      unless ENV["CI"]
+      if ENV["SKIP_DATABASE_RESET"] != "true"
```

Comment:
I could also do

```suggestion
      unless ENV["SKIP_DATABASE_RESET"] == "true"
```

Can't decide which reads better.

Reply:
Or we could say "any value is `true` and only `"false"` or blank is false", meaning this could be:

```suggestion
      if ENV["SKIP_DATABASE_RESET"] == "false"
```

Reply:
Ok, talking it thorough with my self 🦆 I like ensuring a value and then here only honoring the word "false"